maven-jahia-plugin/pom.xml

```diff
@@ -311,19 +311,18 @@
 		<dependency>
 			<groupId>org.apache.felix</groupId>
 			<artifactId>maven-bundle-plugin</artifactId>
-			<version>2.5.3</version>
-			<!-- QA-10988 We didn't manage to upgrade to 2.5.4 because of bugs https://issues.apache.org/jira/browse/FELIX-4556 and https://issues.apache.org/jira/browse/FELIX-4882 -->
+			<version>5.1.2</version>
 			<exclusions>
```

Comment:
I think this exclusion can be removed here, as it was just a workaround for QA-10988, so instead of just removing the commend, the whole exclusion should be removed unless there is again a specific reason for it.